Lib/test/support/socket_helper.py

```diff
@@ -277,3 +278,62 @@ def create_unix_domain_name():
     """
     return tempfile.mktemp(prefix="test_python_", suffix='.sock',
                            dir=os.path.curdir)
+
+
+# consider that sysctl values should not change while tests are running
+_sysctl_cache = {}
+
+def _get_sysctl(name):
+    """Get a sysctl value as an integer."""
+    try:
+        return _sysctl_cache[name]
+    except KeyError:
+        pass
+
+    cmd = ['sysctl', name]
```

Comment:
sysctl provides a `-n` arg to suppress the name:
```
$ sysctl -n net.inet.tcp.blackhole
0
```

Reply:
Interesting, I didn't know it. It's also supported by Linux (even if right now, this function is only called on FreeBSD ;-)). I updated my PR.